src/xdocs/beginning_development.xml

```diff
@@ -23,16 +23,16 @@
         Prepare development environment in Ubuntu</a>.<br/>
         2. Fork Checkstyle upstream project. As it is described
         <a href="https://help.github.com/articles/fork-a-repo/"> here</a><br/>
-        3. Checkout the current source code from: https://github.com/you_user_name/checkstyle/<br/>
-        by running
+        3. Checkout the current source code from: https://github.com/your_user_name/checkstyle/<br/>
+        to get the copy of your fork on your computer, by running
```

Comment:
```suggestion
        to get the copy of your fork on your local, by running
```

We generally use the term "local" to refer to a computer. So yes, IMO local and remote makes more sense than computer and remote :)